src/contracts/misc/GhoCcipSteward.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {EnumerableSet} from '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {IGhoCcipSteward} from './interfaces/IGhoCcipSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+import {UpgradeableLockReleaseTokenPool, RateLimiter} from './deps/Dependencies.sol';
```

Comment:
I d create two files `IUpgradeableLockReleaseTokenPool` and `IRateLimiter` under `dependencies/ccip` directory and state in a dev doc that they are a reduced interfaces with functions needed only. Also a reference to the file in aave/ccip repo.
No need to have the interface with all functions imho.

Reply:
I can do it, but for UpgradeableLockReleaseTokenPool, we actually test most of the functions (or are planning to add them)

Reply:
I did it for UpgradeableLockReleaseTokenPool now. RateLimiter is just a library, so I didn't prefix with "I"